README.md

```diff
@@ -111,7 +111,7 @@ apiVersion: v1
 metadata:
   name: glusterfs-csi-pv
   annotations:
-    volume.beta.kubernetes.io/storage-class: glusterfs-csi
+    volume.kubernetes.io/storage-class: glusterfs-csi
```

Comment:
Since we're fixing the annotation, please switch to using `storageClassName`.

From https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/#using-dynamic-provisioning:
> Before Kubernetes v1.6, this was done via the volume.beta.kubernetes.io/storage-class annotation. However, this annotation is deprecated since v1.6. Users now can and should instead use the storageClassName field of the PersistentVolumeClaim object.